src/macros.jl

```diff
@@ -595,6 +621,21 @@ function subset!_helper(x, args...)
     end
 end
 
+function rsubset!_helper(x, args...)
```

Comment:
Would there be a way to avoid duplicating these helper functions, for example by passing `subset` or `subset!` to a more generic one?

Reply:
Yes, that't what DataFrameMacros does. I will do this in a future PR.

Reply:
Why not do it now? :-p

Reply:
Okay I've simplified the `@byrow` check but it's hard to actually make everything in a coherent function 

1. The ugly `t = (fun_to_vec(ex; no_dest=true, outer_flags=outer_flags) for ex in exprs)` can't be gotten rid of yet because we still need to support some deprecated functionality in `@combine`. 
2. There are lots of exceptions, for example `@subset` requires a keyword argument, `@orderby` doesn't even call a DataFrames function. Writing a function for these cases would be just as easy as writing out the expression.